server/admin/api.go

```diff
@@ -56,6 +56,86 @@ func (s *Server) apiConfig(w http.ResponseWriter, _ *http.Request) {
 	writeJSON(w, s.core.ConfigMsg())
 }
 
+func (s *Server) apiAsset(w http.ResponseWriter, r *http.Request) {
+	assetSymbol := strings.ToLower(chi.URLParam(r, assetSymKey))
+	assetID, found := dex.BipSymbolID(assetSymbol)
+	if !found {
+		http.Error(w, fmt.Sprintf("unknown asset %q", assetSymbol), http.StatusBadRequest)
+		return
+	}
+	asset, err := s.core.Asset(assetID)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("unsupported asset %q / %d", assetSymbol, assetID), http.StatusBadRequest)
+		return
+	}
+
+	var errs []string
+	dexAsset := asset.Asset
+	backend := asset.Backend
+	currentFeeRate, err := backend.FeeRate()
+	if err != nil {
+		errs = append(errs, fmt.Sprintf("unable to get current fee rate: %v", err))
+	}
+
+	synced, err := backend.Synced()
+	if err != nil {
+		errs = append(errs, fmt.Sprintf("unable to get sync status: %v", err))
+	}
+
+	res := &AssetInfo{
+		Asset:          dexAsset,
+		CurrentFeeRate: currentFeeRate,
+		Synced:         synced,
+		Errors:         errs,
+	}
+	writeJSON(w, res)
+}
+
+func (s *Server) apiAssetPOST(w http.ResponseWriter, r *http.Request) {
+	assetSymbol := strings.ToLower(chi.URLParam(r, assetSymKey))
+	assetID, found := dex.BipSymbolID(assetSymbol)
+	if !found {
+		http.Error(w, fmt.Sprintf("unknown asset %q", assetSymbol), http.StatusBadRequest)
+		return
+	}
+	asset, err := s.core.Asset(assetID)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("unsupported asset %q / %d", assetSymbol, assetID), http.StatusBadRequest)
+		return
+	}
+
+	body, err := ioutil.ReadAll(r.Body)
+	r.Body.Close()
+	if err != nil {
+		http.Error(w, fmt.Sprintf("unable to read request body: %v", err), http.StatusInternalServerError)
+		return
+	}
+	if len(body) == 0 {
+		http.Error(w, "no POST data provided", http.StatusBadRequest)
+		return
+	}
+
+	var data AssetPost
+	if err = json.Unmarshal(body, &data); err != nil {
+		http.Error(w, fmt.Sprintf("invalid POST data: %v", err), http.StatusBadRequest)
+		return
+	}
+
+	var settingsFound bool
+
+	if data.FeeRateScale != nil {
+		settingsFound = true
+		log.Infof("Setting %s (%d) fee rate scale factor to %f", strings.ToUpper(assetSymbol), assetID, *data.FeeRateScale)
+		asset.Backend.SetFeeRateScale(*data.FeeRateScale)
```

Comment:
It should be noted that allowing the operator to apply these scale factors means that the operator now shares the blame for stuck swaps with the asset developer. With great power comes great responsibility and great potential for chaos.

Reply:
We could make 1.0 the min

Reply:
Aren't there times when the fees are outrageously high too?

Reply:
For sure.  Just saying it's something we could do out of caution because of the stuck swap risk.  I anticipated setting <1 for cooling down periods like what we've seen recently, but there's certainly a risk if is left <1 without close monitoring.  I think setting <1 could just be met with a warning to the operator.